src/components/Dashboard/SpeechRecognition/SpeechRecognition.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
```

Comment:
Please rewrite your code. We use spaces for identation, not tabs. 1 tab = 2 spaces. https://github.com/airbnb/javascript#whitespace

I suggest you read the entire style guide through.